packages/@aws-cdk/aws-cloudfront-origins/test/integ.origin-group.expected.json

```diff
@@ -72,9 +72,20 @@
             "ForwardedValues": {
               "QueryString": false
             },
-            "TargetOriginId": "cloudfrontorigingroupDistributionOrigin137659A54",
+            "TargetOriginId": "cloudfrontorigingroupDistributionOriginGroup10B57F1D1",
             "ViewerProtocolPolicy": "allow-all"
           },
+          "CacheBehaviors": [
+            {
+              "PathPattern": "/api",
+              "ForwardedValues": {
+                "QueryString": false
+              },
+              "PathPattern": "/api",
+              "TargetOriginId": "cloudfrontorigingroupDistributionOrigin137659A54",
```

Comment:
This should be `cloudfrontorigingroupDistributionOriginGroup10B57F1D1`, not `cloudfrontorigingroupDistributionOrigin137659A54`, correct?

Reply:
Aaah, now I see it. I had assumed it would return an origin instead of an origin group - did not look after the concrete hash. Right!